Alameen688/MyDiary
# #159330749 Client/Postman should be able to delete a specific diary entry for a particular user

#### What does this PR do?
Implements entry deletion by a particular user by using the user id in the token to be provided in the
Authorization header.
#### Description of Task to be completed?
Have the delete entry endpoint working
/DELETE /api/v1/entries/<entry_id>
#### How should this be manually tested?
After cloning the repo, cd into it then
- make sure you have postgresql installed on your machine and start it.
- add a postgreSQL DATABASE_URL to your .env file
   ```
     DATABASE_URL=postgresql://username:password@localhost:5432/mydiarydb
   ```
- The RUN `npm run migrate` to create the database tables.
- Login with an existing user account via /POST api/v1/users/login
  or register at /POST api/v1/users/signup
- send a request to /DELETE /api/v1/entries/<entry_id> where `entry_id` is the id of the entry to be deleted 
- make sure to put the token gotten from logging in earlier in your authorization header in Postman (Client) `"Authorization" : "Bearer <the_token_here>"`
- you should get a It returns a 204 no content response after the post must have been deleted for that user
- And if entry is not found you should get a 404
#### Any background context you want to provide?
none
#### What are the relevant pivotal tracker stories?
#159330749
